xray/audit/jas/jasmanager_test.go

```diff
@@ -37,6 +38,17 @@ var fakeServerDetails = config.ServerDetails{
 	User:     "user",
 }
 
+func initJasTest(t *testing.T, workingDirs ...string) (*AdvancedSecurityScanner, func()) {
+	assert.NoError(t, rtutils.DownloadAnalyzerManagerIfNeeded())
+	scanner, err := NewAdvancedSecurityScanner(workingDirs, &fakeServerDetails)
+	assert.NoError(t, err)
+	return scanner, func() {
+		if scanner.scannerDirCleanupFunc != nil {
```

Comment:
what is reason for it to be nil?

Reply:
same question for all other appearances of if scanner.scannerDirCleanupFunc != nil { 
in the code